src/components/CustomSelect/CustomSelect.tsx

```diff
@@ -56,7 +56,7 @@ const warn = warnOnce("CustomSelect");
 const checkOptionsValueType = (options: CustomSelectOptionInterface[]) => {
   if (new Set(options.map((item) => typeof item.value)).size > 1) {
     warn(
-      "Some values of your options have different types. CustomSelect onChange always returns a string type.",
+      "Некоторые значения ваших опций имеют разные типы. CustomSelect onChange всегда возвращает строковый тип.",
```

Comment:
Кажется, что тут нет смысла дважды повторять про то, что это `CustomSelect`.
```suggestion
      "Некоторые значения ваших опций имеют разные типы. onChange всегда возвращает строковый тип.",
```